### DISCUSSION TITLE
WalletConnect Cloud "Allowed Domains" input will not accept wildcards

### DISCUSSION THREAD

**Discussion by dannyDNS** [WalletConnect/walletconnect-monorepo#4629] [Developer Support]
I set this up previously for the domains that are correctly set in this screenshot but now I cannot add another domain with a wildcard.

![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/129006499/9d03a0c0-0563-4bc5-b60c-290bb22c1c9a)

**Comment by glitch-txs** [comment]
We are working on this and it will be fixed shortly
